erpnext/accounts/doctype/bank_transaction/bank_transaction.py

```diff
@@ -121,7 +125,7 @@ def before_submit(self):
 		self.allocate_payment_entries()
 		self.set_status()
 
-		if frappe.get_single_value("Accounts Settings", "enable_party_matching"):
+		if frappe.db.get_single_value("Accounts Settings", "enable_party_matching"):
```

Comment:
```suggestion
		if frappe.get_single_value("Accounts Settings", "enable_party_matching"):
```

Why should we remove caching here? Seems like an unrelated change.

Reply:
I couldn't find documentation on frappe.get_single_value and exchanged it for the document one

Reply:
`frappe.get_single_value` uses the redis cache, `frappe.db.get_single_value` reads from the database every time.